src/lib/components/event/event-links-expanded.svelte

```diff
@@ -0,0 +1,42 @@
+<script lang="ts">
+  import Link from '$lib/holocene/link.svelte';
+  import type { EventLink } from '$lib/types/events';
+  import { routeForEventHistory } from '$lib/utilities/route-for';
+
+  export let links: EventLink[] = [];
+</script>
+
+{#if links?.length}
```

Comment:
I think this is redundant because `{#each}` won't iterate/render anything for an empty array, but nbd.

Reply:
👍